CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# <a name="contributing">Contributing Overview</a>
+Please do! Thanks for your help improving the project! :balloon:
+
+All contributors are welcome. Please see the [newcomers welcome guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where and why to contribute. This project is community-built and welcomes collaboration. Contributors are expected to adhere to our [Code of Conduct](.CODE_OF_CONDUCT.md).
+
+Not sure where to start? First, see the [newcomers welcome guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit). Grab an open issue with the [help-wanted label](../../labels/help%20wanted) and jump in. Join the [Slack account](http://slack.layer5.io) and engage in conversation. Create a [new issue](/../../issues/new/choose) if needed.  All [pull requests](/../../pulls) should reference an open [issue](/../../issues). Include keywords in your pull request descriptions, as well as commit messages, to [automatically close issues in GitHub](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords).
+
+**Sections**
+- <a name="contributing">General Contribution Flow</a>
+  - <a href="#commit-signing">Developer Certificate of Origin</a>
+- Meshery Contribution Flow
+  - <a href="#contributing-docs">Meshery Documentation</a>
+  - <a href="#contributing-meshery">Meshery Backend</a>
+    - <a href="#adapter">Writing a Meshery Adapter</a>
+  - <a href="#contributing-ui">Meshery UI</a>
+Relevant coding style guidelines are the Go Code Review Comments and the Formatting and style section of Peter Bourgon's Go: Best Practices for Production Environments.
+# <a name="contributing">General Contribution Flow</a>
+
+In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
```

Comment:
I wonder if we can make some of this more generic to the Layer5 community overall.

Reply:
ya good idea we can work on it